code/index.html

```diff
@@ -2,21 +2,139 @@
   </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
+<body>
+
+<!--A header with logo, name, navbar-->
+<header>
+    <div class="logo">
+     <img src="logo.png" alt="logo">
+    </div>
+    <div class="Name">
+    <h2>TAILS</h2>  
+    </div>
+<nav>
+    <ul>
+      <li>Vision</li>
+      <li>Contact</li>
+      <li>Subscribe</li>
+    </ul>
+</nav>
+</header>
+
+<main>
+
+<!--A section for the bigger home image and text-->
+
+<section class="home">
+
+  <div class="big-text">
+    <h1>TAILS</h1>
+    <h2>Contemporary Art Magazine</h2>
+    <h3>Since 2018</h3>
+  </div>
+
+  <div class="big-image">
+    <img src="spiral.gif" alt="spiral"/>
+  </div>
+
+  <div class="big-text">
+    <h2>Interview with Philip Ullman</h2>
+    <h3>About new videowork Diary Room</h3>
+  </div>
+
+  <div class="big-image">
+    <img src="ullman2.png" alt="Diary Room by Philip Ullman"/>
+  </div>
+
+</section>
+
+<!--A section for the regular news articles-->
+
+<section class="news">
+
+<div class="news-box">
+  <img src="Person.png" alt="Person at R1">
+  <h4>Helio by Emmeli Person</h4>
+  <h5>Exhibited at R1, Stockholm</h5>
+</div>
+
+<div class="news-box">
+  <img src="Anjou.jpg" alt="Cosmic Sneeze by Josefina Anjou">
+  <h4>Josefina Anjou</h4>
+  <h5>Cosmic Sneeze at Studio 21, Split</h5>
+</div>
+
+<div class="news-box">
+  <img src="Ullman.png" alt="Philip Ullman at HAUS">
+  <h4>Philip Ullman</h4>
+  <h5>Work in progress at HAUS, Vienna</h5>
+</div>
+
+<div class="news-box">
+  <img src="Hansson.png" alt="Hansson at To the Curious Proceed">
+  <h4>Jonne Hansson</h4>
+  <h5>Heavy Thoughts shown online</h5>
+</div>
+
+<div class="news-box">
+  <img src="Dream Island.png" alt="Dream Island at Art Lab Gnesta">
+  <h4>Launch of Dream Island</h4>
+  <h5>At Art Lab Gnesta, Gnesta</h5>
+</div>
+
+<div class="news-box">
+  <img src="Time Frame.png" alt="Time Frame at Kungstensgatan 21">
+  <h4>Graduation show Time Frame</h4>
+  <h5>Shown at Kungstensgatan 21, Stockholm</h5>
+</div>
+
+<div class="news-box">
+  <img src="Shift Motion.png" alt="Shift Motion at Midsommargården">
+  <h4>Group show Shift Motion</h4>
+  <h5>Shown at Midsommargården, Stockholm</h5>
+</div>
+
+</section>
+
+<section class="big-text">
+
+  <div class="big-image">
+    <img src="emergence.png" alt="emergence"/>
+  </div>
+
+  <div class="big-text">
+    <h5>Credits: Photos by Jacqueline Neubauer & Tor Westerlund. Graphics by Manon Bachelier.</h5>
+  </div>
+
+</section>
+
+</main>
+
+<!--A footer navlinks, logo and Name-->
+
+<footer>
+  <nav>
+    <ul>
+     <li>Vision</li>
+     <li>Contact</li>
+     <li>Subscribe</li>
+   </ul>
+  </nav>
+  <div class="Name">
```

Comment:
I believe you just didn't see this minor thing before hand-in since you formatted your code great elsewhere, but the tab indents should be aligned